riscv-runtime/src/hash.rs

```diff
@@ -27,15 +27,26 @@ pub fn poseidon_gl(data: &mut [Goldilocks; 12]) -> &[Goldilocks; 4] {
 /// Perform one Poseidon2 permutation with 8 Goldilocks field elements in-place.
 pub fn poseidon2_gl_inplace(data: &mut [OpaqueGoldilocks; 8]) {
     unsafe {
-        ecall!(Syscall::Poseidon2GL, in("a0") data, in("a1") data);
+        ecall!(Syscall::Poseidon2GL, in("a0") data, in("a1") data, in("a2") 3);
     }
 }
 
-/// Perform one Poseidon2 permutation with 8 Goldilocks field elements.
-pub fn poseidon2_gl(data: &[OpaqueGoldilocks; 8]) -> [OpaqueGoldilocks; 8] {
+#[repr(u32)]
+pub enum Poseidon2OutputHalf {
+    //None = 0,
+    FirstHalf = 1,
+    SecondHalf = 2,
+    //Full = 3,
+}
```

Comment:
```suggestion
pub enum Poseidon2OutputHalf {
    FirstHalf = 1,
    SecondHalf = 2,
}
```

Reply:
These comments act as a sort of documentation... do you really want them gone?